recipes/ckanweb-deploy-exts.rb

```diff
@@ -89,7 +89,10 @@
 	'odi_certificates' => 20,
 	'dcat structured_data' => 30,
 	'data_qld data_qld_google_analytics' => 40,
-	'scheming_datasets' => 50
+	'scheming_datasets' => 50,
+	'archiver' => 60,
```

Comment:
Is the order of these extensions really important? What breaks if they're in a different order?

Reply:
CKAN ini file changes
ckan.plugins = ...  qa archiver report
(in that specific order)

# Archiver
ckanext-archiver.archive_dir = /app/filestore/archive
ckanext-archiver.cache_url_root = http://dataqld-ckan.docker.amazee.io/resources/
^ These two will need some experimenting in relation to your use of S3 uploads and their settings.

Reply:
The order specified here doesn't match, then; it puts "archiver" first.

Reply:
fixed.